WcaOnRails/db/migrate/20170517213035_update_list_of_countries.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class UpdateListOfCountries < ActiveRecord::Migration[5.0]
+  def up
+    Country.delete_all
+    Country::ALL_STATES.each do |state|
+      Country.create(state)
+    end
+    # These substitions have been found by running the migration and checking
+    # users with iso2 not matching anything in the 'Country' table.
+    {
+      # Virgin Islands -> United Kingdom
+      "VG" => "GB",
+      # Aruba -> Netherlands
+      "AW" => "NL",
+      # French Guinea -> France
+      "GF" => "FR",
+      # Pitcairn Islands -> United Kingdom
+      "PN" => "GB",
+      # Puerto Rico -> USA
+      "PR" => "US",
+      # Isle of Man -> United Kingdom
+      "IM" => "GB",
+      # French Polynesia -> France
+      "PF" => "FR",
+    }.each do |old_iso2, new_iso2|
+      User.where(country_iso2: old_iso2).update(country_iso2: new_iso2)
```

Comment:
This `update` will silently do nothing if validations fail. See the documentation on [Relation update here](https://apidock.com/rails/ActiveRecord/Relation/update). It's probably best to change all calls in this migration from `update` to [`update_all`](https://apidock.com/rails/ActiveRecord/Relation/update_all), which goes directly to the database (skipping validations).